openlibrary/core/waitinglist.py

```diff
@@ -233,7 +243,7 @@ def update_ebook(ebook_key, **data):
         web.ctx.site.store[ebook_key] = dict(ebook2, _rev=None)  # force update
 
 
-def sendmail_book_available(book):
+def sendmail_book_available(book) -> None:  # type: ignore[no-untyped-def]
```

Comment:
I think this might be an edition. Note importing some of our models will very quickly trigger cyclical depencies, so use the `TYPE_CHECKING` guard we have in some other spots.

```suggestion
def sendmail_book_available(book: 'Edition') -> None:
```

Reply:
fixed!